src/features/session/logout/ui/logout-btn.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { logoutModel } from "../model";
+import { Button } from "shared/ui";
+
+export const LogoutButton: React.FC = () => {
+  return (
+    <Button
+      onClick={() => logoutModel.logoutRequested()}
+      sx={{ fontSize: "18px", px: 4, width: 300 }}
```

Comment:
Font size can also be written without px prefix

Reply:
see, have changed it